is_core/forms/widgets.py

```diff
@@ -373,3 +373,33 @@ def render(self, name, value, attrs=None):
     def value_from_datadict(self, data, files, name):
         value = super(MultipleTextInput, self).value_from_datadict(data, files, name)
         return [v.strip() for v in value.split(self.separator)] if isinstance(value, six.string_types) else value
+
+
+class AbstractDateRangeWidget(MultiWidget):
+
+    def __init__(self):
+        class_from, class_to = self.get_range_classes()
+        super(AbstractDateRangeWidget, self).__init__(
+            (
+                TextInput(attrs={'class': class_from}),
+                TextInput(attrs={'class': class_to}),
```

Comment:
tady si jenom hodim do kazdeho ze dvou widgetu prislusnou widget classu pro coffee script, data-filter atribut tady nepridavam protoze pak bych musel pridat jako pozicni argument do konstruktoru 'full_field_name' a o tom ses zminoval, ze to nechces